src/types/package.rs

```diff
@@ -235,7 +238,9 @@ pub struct IssuesListItem {
 pub enum RiskType {
     TotalRisk,
     Vulnerabilities,
-    MaliciousCodeRisk,
+    #[serde(alias = "maliciousRisk")]
+    #[serde(rename = "maliciousCodeRisk")]
+    MaliciousRisk,
```

Comment:
Not related to this PR, but realistically this enum should have the `Risk` suffixes stripped from all variants.

Not sure if we want to immediately do it the right way for the new one considering `Vulnerabilities` is already in the correct format.

Reply:
We should probably do it in a different PR.